lib/opentelemetry-proto/src/convert.rs

```diff
@@ -212,6 +219,46 @@ impl ResourceLog {
             }
         };
 
+        // Insert instrumentation scope (scope name, version, and attributes)
+        if let Some(scope) = self.scope {
+            if !scope.name.is_empty() {
+                log_namespace.insert_source_metadata(
+                    SOURCE_NAME,
+                    &mut log,
+                    Some(LegacyKey::Overwrite(path!(SCOPE_KEY, NAME_KEY))),
+                    path!(SCOPE_KEY, NAME_KEY),
+                    scope.name,
+                );
+            }
+            if !scope.version.is_empty() {
+                log_namespace.insert_source_metadata(
+                    SOURCE_NAME,
+                    &mut log,
+                    Some(LegacyKey::Overwrite(path!(SCOPE_KEY, VERSION_KEY))),
+                    path!(SCOPE_KEY, VERSION_KEY),
+                    scope.version,
+                );
+            }
+            if !scope.attributes.is_empty() {
+                log_namespace.insert_source_metadata(
+                    SOURCE_NAME,
+                    &mut log,
+                    Some(LegacyKey::Overwrite(path!(SCOPE_KEY, ATTRIBUTES_KEY))),
+                    path!(SCOPE_KEY, ATTRIBUTES_KEY),
+                    kv_list_into_value(scope.attributes),
+                );
+            }
+            if scope.dropped_attributes_count > 0 {
```

Comment:
Can we differentiate between `None` scope and an actual `0` value? Is this a possible scenario?

Reply:
If the scope is none (which is possible but not sure under which scenarios), then this entire block should be skipped, creating no attributes under it.

However, `dropped_attributes_count` is [defined as a uint](https://github.com/vectordotdev/vector/blob/master/lib/opentelemetry-proto/src/proto/opentelemetry-proto/opentelemetry/proto/common/v1/common.proto#L80), and defaults to 0... so if `scope` is non-null, then this field should always be present.  I figured no sense in emitting `dropped_attributes_count: 0` for every payload, since this should be the norm.